app/models/test_track/remote/assignment.rb

```diff
@@ -1,10 +1,19 @@
 class TestTrack::Remote::Assignment
-  include TestTrack::RemoteModel
+  include TestTrack::Resource
+  include ActiveModel::Dirty
 
-  attributes :split_name, :variant, :context, :unsynced
+  attribute :split_name
+  attribute :variant
+  attribute :context
+  attribute :unsynced, :boolean
 
   validates :split_name, :variant, :mixpanel_result, presence: true
 
+  def initialize(...)
+    super
+    clear_changes_information
+  end
```

Comment:
This resource uses `ActiveModel::Dirty` to implement `variant_changed?`. It'd probably be better to not rely on AM::Dirty, but I wanted to avoid messing with this.